### PR TITLE
Include updated SXL in the setup package

### PR DESCRIPTION
**Pull request by otterdahl**
Instead of storing the SXL in this repo, build the SXL using a workflow and include in the setup package

### CODE REVIEW

**Comment by emiltin** [comment]
Good idea.
You could also consider including the SXL files as either a git submodule, or as a git subtree.

File: .github/workflows/setup.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Set up ruby
         uses: ruby/setup-ruby@v1
         with:
+          ruby-version: '3.0'
```
Comment (emiltin) [inline .github/workflows/setup.yml:43]: note that latest is 3.2.2

Comment (otterdahl) [reply .github/workflows/setup.yml]: Thanks. It works, however the setup seems much slower

Comment (otterdahl) [reply .github/workflows/setup.yml]: Maybe not that much slower after all

Comment (otterdahl) [reply .github/workflows/setup.yml:36]: Sadly rubyXL doesn't seem to work with ruby 3.2.2 yet. So we need to stick with 3.0 for the time being...

**Pull request merged**